Automattic/wp-calypso
# Exclude wpcom domains from the all domains list

This PR is based on a yet-to-be merged feature in #43699

#### Changes proposed in this Pull Request

* This PR excludes WPCOM domains from the all domains list.

#### Testing instructions



* Visit the [all domains view ](http://calypso.localhost:3000/domains/manage)
* Confirm that WPCOM domains are not listed.
